modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryVersionLocalServiceImpl.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.service.impl;
+
+import com.liferay.object.entry.util.ObjectEntryDTOConverterUtil;
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.model.ObjectEntryVersion;
+import com.liferay.object.service.base.ObjectEntryVersionLocalServiceBaseImpl;
+import com.liferay.object.util.comparator.ObjectEntryVersionVersionComparator;
+import com.liferay.portal.aop.AopService;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.vulcan.dto.converter.DTOConverterRegistry;
+
+import java.util.List;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Feliphe Marinho
+ */
+@Component(
+	property = "model.class.name=com.liferay.object.model.ObjectEntryVersion",
+	service = AopService.class
+)
+public class ObjectEntryVersionLocalServiceImpl
+	extends ObjectEntryVersionLocalServiceBaseImpl {
+
+	@Override
+	public ObjectEntryVersion addObjectEntryVersion(ObjectEntry objectEntry)
+		throws PortalException {
+
+		ObjectEntryVersion objectEntryVersion =
+			objectEntryVersionPersistence.create(
+				counterLocalService.increment());
+
+		int count = objectEntryVersionPersistence.countByObjectEntryId(
+			objectEntry.getObjectEntryId());
+
+		objectEntryVersion.setVersion(++count);
+
+		return _updateObjectEntryVersion(objectEntryVersion, objectEntry);
+	}
+
+	@Override
+	public List<ObjectEntryVersion> getObjectEntryVersions(long objectEntryId) {
+		return objectEntryVersionPersistence.findByObjectEntryId(objectEntryId);
+	}
+
+	@Override
+	public ObjectEntryVersion updateLatestObjectEntryVersion(
+			ObjectEntry objectEntry)
+		throws PortalException {
+
+		ObjectEntryVersion objectEntryVersion =
+			objectEntryVersionPersistence.fetchByObjectEntryId_First(
+				objectEntry.getObjectEntryId(),
+				ObjectEntryVersionVersionComparator.getInstance(false));
+
+		if (objectEntryVersion == null) {
+			return null;
+		}
+
+		return _updateObjectEntryVersion(objectEntryVersion, objectEntry);
+	}
+
+	private ObjectEntryVersion _updateObjectEntryVersion(
+			ObjectEntryVersion objectEntryVersion, ObjectEntry objectEntry)
```

Comment:
Sort it please

Reply:
Done (https://github.com/liferay-bpm/liferay-portal/pull/4218/commits/f001e40c3f252ce8f48e444cb2819b8d6fdd1af2)